src/Test/MakerTestCase.php

```diff
@@ -32,6 +33,10 @@ public function testExecute(MakerTestDetails $makerTestDetails)
 
     protected function executeMakerCommand(MakerTestDetails $testDetails)
     {
+        if (!class_exists(Process::class)) {
+            throw new \LogicException('The MakerTestCase cannot be run as the Process component is not installed. Try running "compose require symfony/process".');
```

Comment:
```suggestion
            throw new \LogicException('The MakerTestCase cannot be run as the Process component is not installed. Try running "compose require --dev symfony/process".');
```

As it's required for tests, I think it's safe to mention the `--dev` flag in the error message.

Reply:
you're right, changed.